witness/src/Dscp/Witness/Launcher/Mode.hs

```diff
@@ -60,21 +68,25 @@ type WitnessWorkMode ctx m =
     , HasLens' ctx MempoolVar
     , HasLens' ctx SDActions
     , HasLens' ctx (KeyResources WitnessNode)
-
+    , HasLens' ctx TxRelayInput
+    , HasLens' ctx TxRelayPipe
     )
 
 ---------------------------------------------------------------------
 -- WorkMode implementation
 ---------------------------------------------------------------------
 
+newtype TxRelayInput = TxRelayInput { getTxRelayInput :: TQueue GTxWitnessed }
```

Comment:
Can you please add comments to these so people don't get confused and don't put random stuff into these queues?